doc/source/man.rst

```diff
@@ -32,206 +32,217 @@ rho performs four major tasks:
 
-Authentication
---------------
+Authentication Profiles
+-----------------------
 
-The first part to configuring rho is setting up authentication credentials. rho uses SSH to connect to the servers on the network, and the username and password or ssh key are used for authentication credentials. An authentication credential is passed when the scan is run by reference in a profile.
+Use the ``rho auth`` command to create and manage authentication profiles.
 
-There can be multiple auth credentials contained in a single profile.
+An authentication profile defines a set of user credentials to be used during a scan. These user credentials include a username and a password or SSH key. Rho uses SSH to connect to servers on the network and uses authentication profiles to obtain the user credentials that are required access those servers.
 
-Creating and Editing Authentication Credentials
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When a scan runs, it uses a network profile that contains the hosts or IP addresses to be accessed. The network profile also contains references to the authentication profiles that are required to access those systems. A single network profile can contain a reference to multiple authentication profiles as needed to connect to all systems in that network.
 
-rho uses SSH credentials to access the servers to get their system information during discovery. These credentials can be either a username-password or username-key pair. Each set of credentials is stored in a separate entry.
+Creating and Editing Authentication Profiles
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-**rho auth add --name=** *name* **--username=** *username* **[--password]** **[--sshkeyfile=** *key_file* **]** **[--sudo-password]** **[--vault=** *vault_file* **]**
+To create an authentication profile, supply SSH credentials as either a username-password pair or a username-key pair. Rho stores each set of credentials in a separate authentication profile entry.
 
-``--name=name``
+**rho auth add --name=** *name* **--username=** *username* **(--password | --sshkeyfile=** *key_file* **)** **[--sudo-password]** **[--vault=** *vault_file* **]**
 
-  This required argument sets the name of the new authentication credentials entry. This should be descriptive, such as identifying the user or server it relates to. For example, ``"server1-rhouser"``. It should never contain the actual password, as this name may be logged or printed during rho execution.
+``--name=name``
 
+  Required. Sets the name of the new authentication profile. Use a descriptive name for the value that is meaningful to your organization. For example, you could identify the user or server that the authentication profile relates to, for example, ``admin12`` or ``server1_jdoe``. Do not include the password as part of the name value, because this name might be logged or printed during rho execution.
```

Comment:
I think I added info here but forgot to click Comment. So, adding again.

Change made:
Required. Sets the name of the new authentication profile. For the value, use a descriptive name that is meaningful to your organization. For example, you could identify the user or server that the authentication profile relates to, such as ``admin12`` or ``server1_jdoe``. Do not include the password as part of this value, because the value for the ``--name`` option might be logged or printed during ``rho`` execution.